### PR TITLE
Minor update to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tkuchida**
Added requirement for developers to ensure their code executes as intended. [ci skip]

### CODE REVIEW

**Comment by chrisdembia** [comment]
LGTM. PRs to change the PR guidelines are not covered in the PR guidelines. How do we proceed??  😱

**Comment by tkuchida** [comment]
> PRs to change the PR guidelines are not covered in the PR guidelines. How do we proceed??

I'm [n'Escher](https://upload.wikimedia.org/wikipedia/en/b/ba/DrawingHands.jpg).

**Comment by aseth1** [comment]
It may also be prudent to add something like:
"if your change introduces options or branches in code execution please verify that they are being tested and that invalid operations throw exceptions"

I am not sure "branches" is the right word, but hopefully the intent is clear.

**Comment by tkuchida** [comment]
> It may also be prudent to add something like:

Added.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ When you are ready to make a PR, please adhere to the following guidelines:
 
 2. Make sure that your request conforms to our [coding standards](#coding-standards).
 
-3. Make sure that tests pass on your local machine before making a pull request. The [README.md](https://github.com/opensim-org/opensim-core) mentions how to run the tests.
+3. Make sure that your code executes as intended and that *all* tests pass on your local machine before making a pull request. The [README.md](https://github.com/opensim-org/opensim-core) mentions how to run the tests. If your changes introduce runtime options or branching in the code, please ensure that all options or branches are being tested and that exceptions are being thrown in invalid scenarios.
```
Comment (aseth1) [inline CONTRIBUTING.md]: :+1: thanks!

Comment (jenhicks) [inline CONTRIBUTING.md]: I would change "mentions" to "explains", but I don't feel strongly. LGTM. Thanks!

**Comment by chrisdembia** [comment]
Since this PR changes the PR guidelines, perhaps we should discuss and merge it in the dev meeting?

**Pull request merged**